### ISSUE TITLE
Can your product provide the feature that create a channel and username with non-english characters?

### ISSUE THREAD

**Issue by tuxxon** [RocketChat/Rocket.Chat.Android#2210]
### Description:

In korea, users want to create a channel and username in korean. so Can you provide the feature that uses UTF-8 characters? So, we can create a channel and username in non-english charaters.

### Steps to reproduce:

1. create a channel
2. type a non-english characters in input box for the channel.

 doesn't work

### Expected behavior:

To create a channel or username in UTF-8 characters.

### Actual behavior:

Doesn't work. but created in english only.

**Comment by lorek123** [comment]
I don't think this is mainly server-side issue, please check if this bug is reported on  https://github.com/RocketChat/Rocket.Chat and if not fill the issue there.
